scripts/app.js

```diff
@@ -11,6 +14,13 @@ const players = [
   },
 ];
 
+let gameData = [
```

Comment:
If the variables is only used in either `config.js` or `game.js` then it's best to define it in those JS files (applies to all variables defined here)

Reply:
fixed